README.md

```diff
@@ -46,7 +46,7 @@ Other Style Guides
   1. [Accessors](#accessors)
   1. [Events](#events)
   1. [jQuery](#jquery)
-  1. [ECMAScript 5 Compatibility](#ecmascript-5-compatibility)
```

Comment:
Previous link to ES5 compatibility table is broken now.
I decided to add lint to ES6 Can I use feature search.